Util/RequestUtils.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the DunglasApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dunglas\ApiBundle\Util;
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Utility functions for working with Symfony HttpFoundation request.
+ *
+ * @author Teoh Han Hui <teohhanhui@gmail.com>
+ */
+abstract class RequestUtils
```

Comment:
The name `Util\RequestUtils` means nothing to me. What does the `getFixedRequest` actually do? Is it a "RequestFactory"? Why do you need to "fix" the request and what do it means?